src/Pos/Launcher/Runner.hs

```diff
@@ -386,38 +444,29 @@ bracketTransport
 bracketTransport tcpAddr =
     bracket (createTransportTCP tcpAddr) (closeTransport)
 
--- | Bracket a transport and use a static set of peers for discovery.
+-- | Bracket a transport for use with a static set of peers (for discovery).
 bracketResources
     :: BaseParams
     -> TCP.TCPAddr
-    -> Set NodeId
-    -> (RealModeResources Production -> Production a)
+    -> (Transport Production -> Production a)
     -> IO a
-bracketResources bp tcpAddr staticPeers action =
+bracketResources bp tcpAddr action =
     loggerBracket (bpLoggingParams bp) .
     runProduction $
         -- Both the DHT and Transport are problematic here:
         -- 1. We assume you'll want a DHT.
         -- 2. We assume your transport takes an IP/port.
```

Comment:
@avieth can you please comment on what did you mean? I see it was introduced in d8618bf9e48f8ade96de0acf069bdf2c2a30e4ba.